src/App.js

```diff
@@ -157,12 +157,12 @@ export default function App() {
               />
 
               {/* /state/ routes are deprecated but still supported. */}
-              <Route exact path="/state/:stateId">
-                <Redirect to="/us/:stateId" />
-              </Route>
-              <Route exact path="/state/:stateId/county/:countyId">
-                <Redirect to="/us/:stateId/county/:countyId" />
-              </Route>
+              <Redirect exact from="/state/:stateId" to="/us/:stateId" />
+              <Redirect
+                exact
+                from="/state/:stateId/county/:countyId"
+                to="/us/:stateId/county/:countyId"
```

Comment:
How do we handle going from the state code (say "/mo" -> "/us/missouri-mo")?

Reply:
That is done in [LocationRouter.tsx#L17](https://github.com/covid-projections/covid-projections/blob/c80e6f6d62448bac9e670745fe80bb1fd20cc3e2/src/screens/LocationPage/LocationRouter.tsx#L17)

Reply:
makes sense! Is there a world in which we could update the stateId here? or is it a problem that it's doing two redirects as opposed to consolidating into one?